bin/fenix-qr-codes.js

```diff
@@ -31,8 +33,21 @@ if (!fs.existsSync(destDir)) {
         `utm_content=${id}`,
         `utm_medium=referral&utm_source=addons.mozilla.org`,
       ].join('&')
-      return QRCode.toFile(filePath, content);
+      return QRCode.toFile(filePath, content, {margin: 1});
     })
   );
+
+  const knownIds = addonIdsWithQRCodes.addonIds;
+  const addonIdsFromAPI = addons.map(({ id }) => id);
+
+  for (const addonId of addonIdsFromAPI) {
+    if (!knownIds.includes(addonId)) {
```

Comment:
This isn't what I wrote in the comment. I was performing a symmetric difference on purpose: we want the two lists to be the same.

Reply:
I should have tested my code, though :p I don't think the snippet in the comment works. Let me fix it in a new PR.

Reply:
Hmm, I was thinking we just needed to check that we have a QR code for every add-on returned from the API, but I guess we also want to make sure our internal list doesn't include any addonIds that are *not* returned by the API. I think I can update my code to do that. I'm not sure we need sets. Couldn't we just sort the two arrays and do a `deepEqual` on them?